packages/sitecore-jss/src/i18n/graphql-dictionary-service.test.ts

```diff
@@ -121,16 +120,21 @@ describe('GraphQLDictionaryService', () => {
       },
     })
       .post('/', /DictionarySearch/gi)
-      .times(2)
       .reply(200, dictionaryQueryResponse);
 
-    const service = new GraphQLDictionaryService({ endpoint, apiKey, siteName });
+    const service = new GraphQLDictionaryService({
+      endpoint,
+      apiKey,
+      siteName,
+      cacheEnabled: true,
+      cacheTimeout: 2,
+    });
 
-    // call fetch twice, and use nock to see if more than 1 request was actually made
-    await service.fetchDictionaryData('en');
-    await service.fetchDictionaryData('en');
-    // eslint-disable-next-line no-unused-expressions
-    expect(nock.isDone()).to.be.false;
+    const result1 = await service.fetchDictionaryData('en');
+    expect(result1).to.have.all.keys('foo', 'bar');
+
+    const result2 = await service.fetchDictionaryData('en');
+    expect(result2).to.have.all.keys('foo', 'bar');
```

Comment:
Before, I was checking that a second http request was not sent via `expect(nock.isDone()).to.be.false;`

Is the updated test still validating that the cache was used instead of making a second http request? How does it know that result2 came from cache?

Reply:
@illiakovalenko It's validated, because I removed `times(2)` function call. In this case if cache will not work and service will fetch twice, then nock should throw network error, because it mock only 1 request